CODEOWNERS

```diff
@@ -1,13 +1,2 @@
-# Org-Level Owners (in alphabetical order)
-* @justincormack 
```

Comment:
Org level owners should be added to code owners as well.

Reply:
Added